### PR TITLE
Update es.js

### PR DESCRIPTION
**Pull request by PaulaRDi18n**
Small changes to provide better information for the users.

### CODE REVIEW

**Comment by luisrudge** [comment]
Thanks for the PR @PaulaRDi18n 🎉  I asked someone that speaks spanish to review this later.

I'm from Florianopolis myself and I couldn't help but notice that you work at resultados digitais. ISs RD using Auth0 for the CMS or something else? Thanks!

**Comment by luisrudge** [comment]
Oh, I just found RDStation's login page and you're using lock. That's amazing! In any case, you should note that when this PR gets merged and released, it will only be available for 11.x versions, and not for 10.x versions. As of today, you're using 10.x:
image

We recently released a new major version with improved security and features and did our best to not introduce any breaking changes. If you need help upgrading, just get in touch via github or ping me directly: luis.rudge@auth0.com

File: src/i18n/es.js [hunk 1]
```diff
@@ -18,11 +18,11 @@ export default {
       password_change_required:
         'Debe actualizar su contraseña porque es la primera vez que ingresa o porque la contraseña está vencida.',
       password_leaked:
-        'Este intento ha sido bloqueado ya que usted utilizó la misma contraseña para registrarse en otra aplicación que tuvo una filtración reciente. Hemos enviado un email con las instrucciones.',
+        'Este intento ha sido bloqueado ya que usted utilizó la misma contraseña para registrarse en otra aplicación que tuvo una filtración reciente de datos. Hemos enviado un email con instrucciones para desbloquear la cuenta.',
       too_many_attempts:
         'Su cuenta ha sido bloqueada luego de múltiples intentos de inicio de sesión consecutivos.',
       'lock.mfa_registration_required':
-        'Por favor enrole su dispositivo antes de continuar con el segundo factor.',
+        'Se requiere autenticación multifactor pero su dispositivo no está registrado. Por favor inscríbalo antes de continuar.',
```
Comment (lbalmaceda) [inline src/i18n/es.js]: I would say `Se requiere autenticación de dos factores. Por favor registre su dispositivo antes de continuar.`

File: src/i18n/es.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default {
   forgotPasswordTitle: 'Restablecer contraseña',
   forgotPasswordAction: '¿Olvidó su contraseña?',
   forgotPasswordInstructions:
-    'Por favor ingrese su dirección de correo. Le enviaremos las instrucciones para restablecer su contrseña.',
+    'Por favor escriba su dirección de correo. Le enviaremos las instrucciones para restablecer su contrseña.',
```
Comment (lbalmaceda) [inline src/i18n/es.js]: `ingrese` is used in the rest of the sentences of this same file. I would keep it.

File: src/i18n/es.js (hunk 1 above)
Comment (PaulaRDi18n) [reply src/i18n/es.js]: Make sense

**Pull request merged**